src/iframe/index.ts

```diff
@@ -0,0 +1,31 @@
+import {PostMessageChannel} from '../rpcAdapter/PostMessageChannel';
+import {APIPublisher} from '../rpcAdapter/APIPublisher';
+import {IFrameController} from './IFrameController';
+
+export type {Commands, Events} from './IFrameController';
+
+const $$PublisherInstanceSymbol = Symbol('$$RPCAPIPublisher');
+
+const onDOMReady = () => {
+    const controller = new IFrameController(globalThis.document.body);
```

Comment:
Why do we use globalThis?

Reply:
The proposal is already live: https://github.com/tc39/proposal-global